public/js/poll/view.js

```diff
@@ -86,9 +86,11 @@
 			poll = View.parseResults(poll);
 			for (var i = 0, l = poll.options.length; i < l; i++) {
 				var option = poll.options[i],
-					optionView = pollView.find('#pollResult' + option.id);
+				    optionView = pollView.find('.poll-view-result[data-poll-result="'+option.id+'"]');
```

Comment:
Looked like I forgot to change that when doing this: https://github.com/schamper/nodebb-plugin-poll/commit/b4a46561aff279e19c23b7c635fda5037c534b84